### PR TITLE
Fixed Help links for  the Circuit Elements

### LINKED ISSUE #3213: Help links in Properties Panel broken
**Describe the bug**
Help links for the circuit elements in the properties panel are not working.
This is because of the change in the file structure of the CircuitVerseDocs repository.

**To Reproduce**
1. Go to `simulator`
2. Click on any circuit element on canvas
3. Click Help button in Properties Panel

**Expected behavior**
Fix them with the proper links.

**Screenshots**

https://user-images.githubusercontent.com/61665451/175778181-02286392-3f3a-4a98-b311-7669843bee1b.mp4

**Device Information [optional]:**

**Are you working on this issue? (Yes/No)**

### PR DESCRIPTION
**Pull request by Prerna-0202**
Fixes #3213 

#### Describe the changes you have made in this PR -
I have corrected the help links for all the circuit elements and removed the repeating lines of code in 'flag.js'  some lines of code are repeating twice.

### Screenshots of the changes (If any) -
https://user-images.githubusercontent.com/89515816/175788645-cdc0474a-90db-4fdc-af2d-c56aa027328d.mov

Before:-
Screenshot (476)

After:-
Screenshot (477)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by devartstar** [comment]
Nicely done!
Can you please update your branch with the latest master, that would fix the one failing check.

File: simulator/src/modules/Flag.js [hunk 1]
```diff
@@ -205,16 +205,7 @@ export default class Flag extends CircuitElement {
...
     "FLag ToolTip: Use this for debugging and plotting.";
 Flag.prototype.helplink =
-    "https://docs.circuitverse.org/#/timing_diagrams?id=using-flags";
-
-/**
- * @memberof Flag
- * Help URL
- * @type {string}
- * @category modules
- */
```
Comment (tachyons) [inline simulator/src/modules/Flag.js:215]: Why is this removed?, this is for jsdoc

Comment (Prerna-0202) [reply simulator/src/modules/Flag.js:215]: This jsdoc is for this link `Flag.prototype.helplink ="https://docs.circuitverse.org/#/miscellaneous?id=tunnel";` which is the redundant code in this file, so I removed that link, and I think If the link is removed so the jsdoc for that is also of no use, that's why I removed it . I have attached the screenshot also.

Comment (tachyons) [reply simulator/src/modules/Flag.js:215]: But that will be applicable for the helplink below, So keep the jsdoc and just remove the redundant link

Comment (Prerna-0202) [reply simulator/src/modules/Flag.js]: Done @tachyons

**Comment by tachyons** [comment]
Great :shipit:

**Pull request merged**